docs/reference/snapshot-restore/apis/delete-snapshot-api.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[delete-snapshot-api]]
+=== Delete snapshot API
+++++
+<titleabbrev>Delete snapshot</titleabbrev>
+++++
+
+Deletes a <<snapshot-restore,snapshot>> of a cluster or specified data streams and
+indices.
+
+////
+[source,console]
+----
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+----
+// TESTSETUP
+////
```

Comment:
You need to create the `my_snapshot`, `snapshot_2`, and `snapshot_3` snapshots so you can delete them later.
```suggestion

PUT /_snapshot/my_repository/my_snapshot?wait_for_completion=true

PUT /_snapshot/my_repository/snapshot_2?wait_for_completion=true

PUT /_snapshot/my_repository/snapshot_3?wait_for_completion=true
----
// TESTSETUP
////
```

Reply:
Ahhh, that was the issue. I knew that something was wrong with the TESTSETUP but couldn't pinpoint the issue. Thanks!